server/src/routes/shift.js

```diff
@@ -0,0 +1,33 @@
+import express from "express";
+import { checkToken } from "../util/middleware.js";
+import * as validator from "../util/shiftMiddleware.js";
+import * as controller from "../controller/shiftController.js";
+import * as express_validator from "express-validator";
+
+const body = express_validator.body;
+const param = express_validator.param;
+
+const router = express.Router();
+
+router.post(
+  "/",
+  validator.isShiftId,
```

Comment:
You won't have a shiftId yet

Reply:
replaced: 

router.post(
  "/",
  // validator.isShiftId, // You won't have a shiftId yet
  controller.createShift
);

Reply:
Fixed